ocs_ci/ocs/ui/views.py

```diff
@@ -1135,6 +1131,15 @@
         "//p[@data-test='pool-bound-message']",
         By.XPATH,
     ),
+    "used_raw_capacity_in_UI": ("//div[@class='ceph-raw-card-legend__text']", By.XPATH),
+    "actions_inside_pool": (
+        "//div[@class='pf-c-dropdown pf-m-align-right']",
```

Comment:
This locator also shouldn't be working in 4.16 and I don't see it used anywhere

Reply:
This is used in older versions here https://github.com/red-hat-storage/ocs-ci/blob/354fc2328a51c631aab7c0f91b72507c2b6bb6b4/ocs_ci/ocs/ui/block_pool.py#L90
and the code is also working for 4.16 as well I have checked.